kahuna/public/js/usage-rights/usage-rights-editor.js

```diff
@@ -187,6 +183,8 @@ usageRightsEditor.controller(
     function uiError(error) {
         ctrl.error = error.body.errorMessage;
     }
+
+    init();
```

Comment:
This means we will be doing a call to the API twice which isn't needed.

We could either memoize the categories in the service, or have two methods here, `fetchCategories` & `setCategory`.